plugins/ssm-secret.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: ssm-secret
+spec:
+  version: v1.2.1
+  homepage: https://github.com/pr8kerl/kubectl-ssm-secret
+  caveats: |
+    Read the documentation at https://github.com/pr8kerl/kubectl-ssm-secret.
+    You can call this plugin with "kubectl ssm-secret".
```

Comment:
> You can call this plugin with "kubectl ssm-secret".

This does not seem to add anything useful. If you support the `--help` flag, you could say `kubectl ssm-secret --help`.

Reply:
Added in the help flag. Yep - makes more sense. Tx.